javascript-algorithms-and-data-structures/pokemon-search-app/styles.css

```diff
@@ -0,0 +1,135 @@
+body {
```

Comment:
The project lacks responsiveness on smaller screens too.

Reply:
Thank you Lary for having a look into this, I have made these changes in the latest commit!